diff --git a/components/esp_hw_support/clk_ctrl_os.c b/components/esp_hw_support/clk_ctrl_os.c
index a970b4e8b20..6fe1b6c5473 100644
--- a/components/esp_hw_support/clk_ctrl_os.c
+++ b/components/esp_hw_support/clk_ctrl_os.c
@@ -24,7 +24,7 @@
             spin_unlock_irqrestore((lock), g_flags); \
             g_flags = UINT32_MAX; \
         } while(0)
-#define LOCK_INITIALIZER_UNLOCKED       0
+#define LOCK_INITIALIZER_UNLOCKED       SP_UNLOCKED
 
 typedef spinlock_t lock_type_t;
 static irqstate_t g_flags = UINT32_MAX;
diff --git a/components/esp_hw_support/modem_clock.c b/components/esp_hw_support/modem_clock.c
index 376be6d3b9a..2b20cf8cc73 100644
--- a/components/esp_hw_support/modem_clock.c
+++ b/components/esp_hw_support/modem_clock.c
@@ -32,7 +32,7 @@
             spin_unlock_irqrestore((lock), g_flags); \
             g_flags = UINT32_MAX; \
         } while(0)
-#define LOCK_INITIALIZER_UNLOCKED       0
+#define LOCK_INITIALIZER_UNLOCKED       SP_UNLOCKED
 
 typedef spinlock_t lock_type_t;
 static irqstate_t g_flags = UINT32_MAX;
